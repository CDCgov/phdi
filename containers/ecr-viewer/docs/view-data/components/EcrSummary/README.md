[**ecr-viewer**](../../../README.md) â€¢ **Docs**

***

[ecr-viewer](../../../README.md) / view-data/components/EcrSummary

# view-data/components/EcrSummary

## Index

### Functions

- [default](functions/default.md)
