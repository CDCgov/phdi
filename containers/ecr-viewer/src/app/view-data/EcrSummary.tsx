import {evaluate} from "fhirpath";
import {Bundle} from "fhir/r4";

interface EcrViewerProps {
    fhirPathMappings: PathMappings
    fhirBundle: Bundle
}

interface PathMappings {
    [key: string]: string;
}

const patientAddress = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const streetAddresses = evaluate(fhirBundle, fhirPathMappings.patientStreetAddress).join("\n");
    const city = evaluate(fhirBundle, fhirPathMappings.patientCity);
    const state = evaluate(fhirBundle, fhirPathMappings.patientState);
    const zipCode = evaluate(fhirBundle, fhirPathMappings.patientZipCode);
    const country = evaluate(fhirBundle, fhirPathMappings.patientCountry);
    return(
    `${streetAddresses}
    ${city}, ${state}
    ${zipCode}${country && `, ${country}`}`);
}

const patientName = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const givenNames = evaluate(fhirBundle, fhirPathMappings.patientGivenName).join(" ");
    const familyName = evaluate(fhirBundle, fhirPathMappings.patientFamilyName);

    return `${givenNames} ${familyName}`;
}

const patientContactInfo = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const phoneNumbers = evaluate(fhirBundle, fhirPathMappings.patientPhoneNumbers).map(phoneNumber => `${phoneNumber?.use?.charAt(0).toUpperCase() + phoneNumber?.use?.substring(1)} ${phoneNumber.value}`).join("\n");
    const emails = evaluate(fhirBundle, fhirPathMappings.patientEmails).map(email => `${email.value}`).join("\n");

    return `${phoneNumbers}
    ${emails}`
}

const EcrSummary = (
    {fhirPathMappings, fhirBundle}: EcrViewerProps
) => {
    const rr = evaluate(fhirBundle, fhirPathMappings.rrSection)[0];
    return (
        <div className={"info-container"}>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Name</h4></div>
                            <div className="grid-col-auto">
                                {patientName(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>DOB</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Address</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {patientAddress(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Contact</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {patientContactInfo(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                    </div>
                </div>
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Condition
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Reportable Condition</h4></div>
                            <div className="grid-col-fill">
                                {rr.entry[0].display.split("-")[1]}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>RCKMS Trigger Summary</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                                <h4>Lab results relevant to reportable condition</h4>
                            <div className="usa-card__container margin-left-0 margin-right-0 margin-top-1">
                                <div className="usa-card__header padding-top-2 padding-bottom-2 padding-left-3">
                                    <p>
                                        Hepatitis C IgG w/Rfx PCR (temp)<span
                                        className="usa-tag margin-left-1 bg-error-dark text-white">Abnormal</span>
                                    </p>
                                </div>
                                <div className={"card__line "}/>
                                <div className="usa-card__body padding-0">
                                    <table className="usa-table usa-table--borderless ecrTable">
                                        <thead>
                                        <tr>
                                            <th scope="col">Component</th>
                                            <th scope="col">Value</th>
                                            <th scope="col">Ref Range</th>
                                            <th scope="col">Specimen</th>
                                        </tr>
                                        </thead>
                                        <tbody>
                                        <tr>
                                            <td>Hepatitis C Antibody (temp)</td>
                                            <td>Positive (A) (temp)</td>
                                            <td>Not Detected (temp)</td>
                                            <td>Blood (temp)</td>
                                        </tr>
                                        </tbody>
                                    </table>
                                    <div className={"card__line"}/>
                                    {/*todo: make a class for these*/}
                                    <div
                                        className={"padding-left-3 padding-right-3 padding-top-205 padding-bottom-205"}>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Analysis time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                N/A (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Collection time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                05/12/2022 6:00 AM CDT (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Received time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                05/12/2022 11:30 AM CDT (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Notes</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                A detected result is positive and indicates the presence of the virus in the sample. A not detected result indicates that the test did not detect the virus in the sample. This test was performed using a multiplexed nucleic acid amplification test that is labeled Emergence Use Only by the U.S. FDA. Results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. (temp)
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <h4>Clinical sections relevant to reportable condition</h4>
                            <div className={"padding-top-05"}>No matching clinical data found in this eCR (temp) </div>
                        </div>
                    </div>
                </div>

            </div>
        </div>);
};

export default EcrSummary;