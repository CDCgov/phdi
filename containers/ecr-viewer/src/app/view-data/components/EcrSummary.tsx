import {evaluate} from "fhirpath";
import {Bundle} from "fhir/r4";
import {PathMappings, formatPatientName, formatPatientAddress, formatPatientContactInfo} from "../../utils";

interface EcrViewerProps {
    fhirPathMappings: PathMappings
    fhirBundle: Bundle
}


const EcrSummary = (
    {fhirPathMappings, fhirBundle}: EcrViewerProps
) => {
    return (
        <div className={"info-container"}>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Name</h4></div>
                            <div className="grid-col-fill">
                                {formatPatientName(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>DOB</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Address</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {formatPatientAddress(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Contact</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {formatPatientContactInfo(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                    </div>
                </div>
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Condition
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Reportable Condition</h4></div>
                            <div className="grid-col-fill">
                                {evaluate(fhirBundle, fhirPathMappings.rrDisplayName)[0].split('-')[1]}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>RCKMS Trigger Summary</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.rckmsTriggerSummary)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                                <h4>Lab results relevant to reportable condition</h4>
                            <div className="usa-card__container margin-left-0 margin-right-0 margin-top-1">
                                <div className="usa-card__header padding-top-2 padding-bottom-2 padding-left-3">
                                    <p>
                                        Hepatitis C IgG w/Rfx PCR (temp)<span
                                        className="usa-tag margin-left-1 bg-error-dark text-white">Abnormal</span>
                                    </p>
                                </div>
                                <div className={"card__line "}/>
                                <div className="usa-card__body padding-0">
                                    <table className="usa-table usa-table--borderless ecrTable">
                                        <thead>
                                        <tr>
                                            <th scope="col">Component</th>
                                            <th scope="col">Value</th>
                                            <th scope="col">Ref Range</th>
                                            <th scope="col">Specimen</th>
                                        </tr>
                                        </thead>
                                        <tbody>
                                        <tr>
                                            <td>Hepatitis C Antibody (temp)</td>
                                            <td>Positive (A) (temp)</td>
                                            <td>Not Detected (temp)</td>
                                            <td>Blood (temp)</td>
                                        </tr>
                                        </tbody>
                                    </table>
                                    <div className={"card__line"}/>
                                    <div
                                        className={"padding-left-3 padding-right-3 padding-top-205 padding-bottom-205"}>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Analysis time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                N/A (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Collection time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                05/12/2022 6:00 AM CDT (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Received time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                05/12/2022 11:30 AM CDT (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Notes</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                A detected result is positive and indicates the presence of the virus in
                                                the sample. A not detected result indicates that the test did not detect
                                                the virus in the sample. This test was performed using a multiplexed
                                                nucleic acid amplification test that is labeled Emergence Use Only by
                                                the U.S. FDA. Results of this test should not be used as the sole basis
                                                for diagnosis, treatment, or other management decisions. (temp)
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <h4>Clinical sections relevant to reportable condition</h4>
                            <div className={"padding-top-05"}>No matching clinical data found in this eCR (temp) </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>);
};

export default EcrSummary;