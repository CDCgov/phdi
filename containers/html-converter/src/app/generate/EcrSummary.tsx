import {evaluate} from "fhirpath";
import fs from "fs";
import {parse} from "yaml";
import {Bundle} from "fhir/r4";

interface EcrViewerProps {
    fhirBundle: any
}

const EcrSummary = (
    // {fhirBundle}: EcrViewerProps
) => {
    const file = fs.readFileSync('./src/app/generate/fhirPath.yml', 'utf8').toString();
    const fhirBundle: Bundle = JSON.parse(fs.readFileSync('./src/app/generate/exampleBundle.json', 'utf8').toString());
    const fhirPathMappings = parse(file);
    return (
        <div>
            <h1>Quick eCR Summary</h1>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3
                        className="usa-summary-box__heading padding-top-105 padding-bottom-1"
                        id="summary-box-key-information"
                    >
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="grid-col-2 text-bold">Patient Name</div>
                            <div className="grid-col-auto">
                                {evaluate(fhirBundle, fhirPathMappings.patientGivenName).join(" ")} {evaluate(fhirBundle, fhirPathMappings.patientFamilyName)}
                            </div>
                        </div>
                        <div className={"section__line"} />
                        <div className="grid-row">
                            <div className="grid-col-2 text-bold">Patient Address</div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientStreetAddress).join("\n")} {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientCity)}, {evaluate(fhirBundle, fhirPathMappings.patientState)} {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientZipCode)}, USA
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                        <div className="grid-col-2 text-bold">Patient Contact</div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientPhoneNumbers).map(phoneNumber => {
                                    return `tel: (${phoneNumber.use}) ${phoneNumber.value}`
                                }).join("\n")}
                                {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientEmails).map(email => {
                                    return `email: ${email.value}`
                                }).join("\n")}
                            </div>
                        </div>
                        <div className={"section__line"} />
                    </div>
                </div>
            </div>
        </div>);
};

export default EcrSummary;