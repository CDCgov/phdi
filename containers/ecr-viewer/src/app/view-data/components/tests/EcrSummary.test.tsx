import { render } from "@testing-library/react";
import { axe } from "jest-axe";
import EcrSummary from "../EcrSummary";

describe("EcrSummary", () => {
  let container: HTMLElement;

  const patientDetails = [
    {
      title: "Patient Name",
      value: "ABEL CASTILLO",
    },
    {
      title: "DOB",
      value: "04/15/2015",
    },
    {
      title: "Patient Address",
      value: "1050 CARPENTER ST\nEDWARDS, CA\n93523-2800, US",
    },
    {
      title: "Patient Contact",
      value: "Home (818)419-5968\nMELLY.C.A.16@GMAIL.COM",
    },
  ];
  const encounterDetails = [
    {
      title: "Facility Name",
      value: "PRM- Palmdale Regional Medical Center",
    },
    {
      title: "Facility Address",
      value: "38600 Medical Center Drive\nPalmdale, CA\n93551, USA",
    },
    {
      title: "Facility Contact",
      value: "(661)382-5000",
    },
    {
      title: "Encounter Date/Time",
      value: "Start: 05/13/2022 7:25 AM UTC\nEnd: 05/13/2022 9:57 AM UTC",
    },
    {
      title: "Encounter Type",
      value: "Emergency",
    },
  ];
  const aboutTheCondition = [
    {
      title: "Reportable Condition",
      value: "Influenza caused by Influenza A virus subtype H5N1 (disorder)",
    },
    {
      title: "RCKMS Rule Summary",
      value: "Cough",
    },
  ];

  beforeAll(() => {
    container = render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        aboutTheCondition={aboutTheCondition}
      />,
    ).container;
  });
  it("should match snapshot", () => {
    expect(container).toMatchSnapshot();
  });
  it("should pass accessibility test", async () => {
    expect(await axe(container)).toHaveNoViolations();
  });
});
