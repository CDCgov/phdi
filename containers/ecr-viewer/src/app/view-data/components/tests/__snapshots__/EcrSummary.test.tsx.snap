// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EcrSummary should match snapshot 1`] = `
<div>
  <div
    class="info-container"
  >
    <div
      class="usa-summary-box padding-3"
    >
      <div>
        <h3>
          About the Patient
        </h3>
        <div
          class="usa-summary-box__text"
        >
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Patient Name
              </h4>
            </div>
            <div
              class="grid-col-fill"
            >
              ABEL CASTILLO
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                DOB
              </h4>
            </div>
            <div
              class="grid-col-fill text-pre-line"
            >
              2015-04-15
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Patient Address
              </h4>
            </div>
            <div
              class="grid-col-fill text-pre-line"
            >
              1050 CARPENTER ST
    EDWARDS, CA
    93523-2800, US
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Patient Contact
              </h4>
            </div>
            <div
              class="grid-col-fill text-pre-line"
            >
              Home +18184195968
    MELLY.C.A.16@GMAIL.COM
            </div>
          </div>
          <div
            class="section__line"
          />
        </div>
        <div
          class="usa-summary-box__body"
        >
          <h3
            id="summary-box-key-information"
          >
            About the Encounter
          </h3>
        <div
          class="usa-summary-box__text"
        >
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Facility Name
              </h4>
            </div>
            <div
              class="grid-col-auto"
            >
              PRM- Palmdale Regional Medical Center
            </div>
          </div>
          <div
            class="section__line"
          >
        </div>
        <div
          class="grid-row"
        >
          <div
            class="data-title"
          >
          <h4>
            Facility Address
          </h4>
        </div>
        <div
          class="grid-col-auto
          text-pre-line"
        >
          38600 Medical Center Drive
          Palmdale, CA
          93551, USA
        </div>
      </div>
      <div
        class="section__line"
      >
      </div>
      <div
        class="grid-row"
      >
        <div
          class="data-title"
        >
          <h4>
            Facility Contact
          </h4>
        </div>
        <div
          class="grid-col-auto"
        >
          (661)382-5000
        </div>
      </div>
      <div
        class="section__line"
      >
      </div>
      <div
        class="grid-row"
      >
        <div
          class="data-title"
        >
      <h4>
        Encounter Date
      </h4>
      </div>
      <div
        class="grid-col-auto"
      >
        4/27/2022
      </div>
    </div>
    <div
      class="section__line"
    >
    </div>
    <div
      class="grid-row"
    >
      <div
        class="data-title"
      >
        <h4>
          Encounter Type
        </h4>
      </div>
      <div
        class="grid-col-auto"
      >
        Emergency
      </div>
      <div
        class="usa-summary-box__body"
      >
        <h3
          id="summary-box-key-information"
        >
          About the Encounter
        </h3>
        <div
          class="usa-summary-box__text"
        >
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Facility Name
              </h4>
            </div>
            <div
              class="grid-col-auto"
            >
              PRM- Palmdale Regional Medical Center
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Facility Address
              </h4>
            </div>
            <div
              class="grid-col-auto text-pre-line"
            >
              38600 Medical Center Drive
    Palmdale, CA
    93551, USA
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Facility Contact
              </h4>
            </div>
            <div
              class="grid-col-auto"
            >
              (661)382-5000
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Encounter Date
              </h4>
            </div>
            <div
              class="grid-col-auto"
            >
              5/13/2022
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Encounter Type
              </h4>
            </div>
            <div
              class="grid-col-auto"
            >
              Emergency
            </div>
          </div>
        </div>
      </div>
      <div
        class="usa-summary-box__body"
      >
        <h3
          class="margin-bottom-105 margin-top-205"
        >
          About the Condition
        </h3>
        <div
          class="usa-summary-box__text"
        >
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                Reportable Condition
              </h4>
            </div>
            <div
              class="grid-col-fill"
            >
               Influenza caused by Influenza A virus subtype H5N1 (disorder)
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <div
              class="data-title"
            >
              <h4>
                RCKMS Trigger Summary
              </h4>
            </div>
            <div
              class="grid-col-fill text-pre-line"
            >
              Detection of Influenza A nucleic acid in a clinical specimen by any method, including RT-PCR and rapid molecular assays
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <h4>
              Lab results relevant to reportable condition
            </h4>
            <div
              class="usa-card__container margin-left-0 margin-right-0 margin-top-1"
            >
              <div
                class="usa-card__header padding-top-2 padding-bottom-2 padding-left-3"
              >
                <p>
                  Hepatitis C IgG w/Rfx PCR (temp)
                  <span
                    class="usa-tag margin-left-1 bg-error-dark text-white"
                  >
                    Abnormal
                  </span>
                </p>
              </div>
              <div
                class="card__line "
              />
              <div
                class="usa-card__body padding-0"
              >
                <table
                  class="usa-table usa-table--borderless ecrTable"
                >
                  <thead>
                    <tr>
                      <th
                        scope="col"
                      >
                        Component
                      </th>
                      <th
                        scope="col"
                      >
                        Value
                      </th>
                      <th
                        scope="col"
                      >
                        Ref Range
                      </th>
                      <th
                        scope="col"
                      >
                        Specimen
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        Hepatitis C Antibody (temp)
                      </td>
                      <td>
                        Positive (A) (temp)
                      </td>
                      <td>
                        Not Detected (temp)
                      </td>
                      <td>
                        Blood (temp)
                      </td>
                    </tr>
                  </tbody>
                </table>
                <div
                  class="card__line"
                />
                <div
                  class="padding-left-3 padding-right-3 padding-top-205 padding-bottom-205"
                >
                  <div
                    class="grid-row"
                  >
                    <div
                      class="data-title"
                    >
                      <h4>
                        Analysis time
                      </h4>
                    </div>
                    <div
                      class="grid-col-fill text-pre-line"
                    >
                      N/A (temp)
                    </div>
                  </div>
                  <div
                    class="section__line_gray"
                  />
                  <div
                    class="grid-row"
                  >
                    <div
                      class="data-title"
                    >
                      <h4>
                        Collection time
                      </h4>
                    </div>
                    <div
                      class="grid-col-fill text-pre-line"
                    >
                      05/12/2022 6:00 AM CDT (temp)
                    </div>
                  </div>
                  <div
                    class="section__line_gray"
                  />
                  <div
                    class="grid-row"
                  >
                    <div
                      class="data-title"
                    >
                      <h4>
                        Received time
                      </h4>
                    </div>
                    <div
                      class="grid-col-fill text-pre-line"
                    >
                      05/12/2022 11:30 AM CDT (temp)
                    </div>
                  </div>
                  <div
                    class="section__line_gray"
                  />
                  <div
                    class="grid-row"
                  >
                    <div
                      class="data-title"
                    >
                      <h4>
                        Notes
                      </h4>
                    </div>
                    <div
                      class="grid-col-fill text-pre-line"
                    >
                      A detected result is positive and indicates the presence of the virus in the sample. A not detected result indicates that the test did not detect the virus in the sample. This test was performed using a multiplexed nucleic acid amplification test that is labeled Emergence Use Only by the U.S. FDA. Results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. (temp)
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="section__line"
          />
          <div
            class="grid-row"
          >
            <h4>
              Clinical sections relevant to reportable condition
            </h4>
            <div
              class="padding-top-05"
            >
              No matching clinical data found in this eCR (temp) 
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
</div>
`;
